src/element-service.js

```diff
@@ -63,6 +65,19 @@ export function getElementServiceIfAvailable(win, id, extension, opt_element) {
   if (s) {
     return /** @type {!Promise<?Object>} */ (s);
   }
+  /**
+   * If there is (or was) a stubbed extension wait for it to load before trying
+   * to get the service.  Prevents a race condition when everything but
+   * the extensions is in cache.  If there is no stub then it's either loaded,
```

Comment:
Correct. The only ones I could find that were not module level were `amp-geo` and `amp-experiment` (experiment doesn't need an urgent fix because it's render blocking but we should fix it).